index.html

```diff
@@ -31,5 +31,124 @@ <h1>Hey there. I’m Santiago.</h1>
             </ul>
         </div>
     </section>
+    <section class="works">
+        <div class="works-title">
+            <div><img></div>
+            <h2>Works</h2>
+            <div><img></div>
+        </div>
+        <div class="works-card">
+            <div>
+                <h3>Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
```

Comment:
Kindly add the images on the Figma template it will enable us to make room to insert your projects images later

![image](https://user-images.githubusercontent.com/43564324/158901894-3e52ebf9-01b3-41f6-b927-a88f5aabf6a0.png)